### ISSUE TITLE
training time

### ISSUE THREAD

**Issue by rshivansh** [palver7/EquiConvPytorch#1]
Hi, thanks for the code. I am using equi_conv in my own code to train resnet50 on custom data. however I find it is really slow, so  I wanted to check with you. Is that the case?

**Comment by palver7** [comment]
@rshivansh Yes, the code is very slow. If you want to optimize it, you have to create a lookup table for the offsets at each layer of your model.
